micro-rdk/src/esp32/exec.rs

```diff
@@ -2,55 +2,47 @@
 use crate::common::webrtc::exec::WebRtcExecutor;
 use async_executor::{LocalExecutor, Task};
 use futures_lite::{
-    future::{self},
+    future::{self, block_on},
     Future,
 };
-use std::rc::Rc;
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 /// This executor is local and bounded to the CPU that created it usually you would create it after spwaning a thread on a specific core
-pub struct Esp32Executor<'a> {
-    /// A local executor
-    executor: Rc<LocalExecutor<'a>>,
+pub struct Esp32Executor {}
```

Comment:
Is the above comment accurate? I don't see anything in the docs for `LocalExecutor` that says it is bound to a specific core. It is bound to the creating thread.

Reply:
I removed it